ui/perfherder/compare/CompareTable.jsx

```diff
@@ -13,12 +13,45 @@ import JobModel from '../../models/job';
 import SimpleTooltip from '../../shared/SimpleTooltip';
 import { displayNumber } from '../helpers';
 import { compareTableText } from '../constants';
+import { getHashSortOrder } from '../../helpers/sort';
 import ProgressBar from '../ProgressBar';
 import { hashFunction } from '../../helpers/utils';
 
 import TableAverage from './TableAverage';
 
 export default class CompareTable extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      sortedData: [],
+      sorted: false,
+      sortedTestName: '',
+      sortedColumnName: '',
+      sortOrder: false,
+      sortClass: '',
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ sortedData: [...this.props.data] });
```

Comment:
You can probably do this directly in the state rather than in `componentDidMount`